numpy/lib/npyio.py

```diff
@@ -327,6 +327,9 @@ def load(file, mmap_mode=None, allow_pickle=False, fix_imports=True,
         If ``allow_pickle=True``, but the file cannot be loaded as a pickle.
     ValueError
         The file contains an object array, but ``allow_pickle=False`` given.
+    EOFError
+        When calling `np.load` multiple times on the same file handle,
```

Comment:
```suggestion
        When calling ``np.load`` multiple times on the same file handle,
```
probably works as a link, but we are already in the function anyway :).